### PR TITLE
Dwf 2018 12 05 seifried

### PR DESCRIPTION
**Pull request by kurtseifried**

### CODE REVIEW

File: 2018/1000xxx/CVE-2018-1000853.json [hunk 1]
```diff
@@ -0,0 +1 @@
...cription": {         "description_data": [             {                 "lang": "eng",                 "value": "gogs version prior to version 0.11.69.1024 contains a Directory Traversal vulnerability in session  management: file-based session manager (vendor/github.com/go-macaron/session/session.go) that can result in attacker can  authenticate as an arbitrary account; RCE possible if account is an admin. This attack appear to be exploitable via attacker  crafts a malicious session cookie pointing to a file they uploaded."             }         ]     },     "problemtype": {         "problemtype_data": [             {                 "description": [                     {                         "lang": "eng",                         "value": "Directory Traversal"                     }                 ]             }         ]     },     "references": {         "reference_data": [             {                 "url": "https://github.com/gogs/gogs/issues/5469"             },             {                 "url": "https://github.com/go-gitea/gitea/issues/5140"             },             {                 "url": "https://www.anquanke.com/post/id/163575"             }         ]     } }
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000853.json]: This is a reservation duplicate of CVE-2018-18925; we have rejected it.

File: 2018/1000xxx/CVE-2018-1000859.json [hunk 2]
```diff
@@ -0,0 +1 @@
...                         ]                     },                     "vendor_name": "PHP"                 }             ]         }     },     "data_format": "MITRE",     "data_type": "CVE",     "data_version": "4.0",     "description": {         "description_data": [             {                 "lang": "eng",                 "value": "PHP version  7.3.0RC6 contains a CWE-78 (Command injection) vulnerability in imap_open function that  can result in OS access via PHP application, often as root. This attack appear to be exploitable via Depending on  implementation, often an authenticated PHP session."             }         ]     },     "problemtype": {         "problemtype_data": [             {                 "description": [                     {                         "lang": "eng",                         "value": "CWE-78 (Command injection)"                     }                 ]             }         ]     },     "references": {         "reference_data": [             {                 "url": "https://github.com/rapid7/metasploit-framework/pull/10987"             },             {                 "url": "https://antichat.com/threads/463395"             }         ]     } }
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000859.json]: This is a reservation duplicate of CVE-2018-19518; we have rejected it.

File: 2018/1000xxx/CVE-2018-1000853.json (hunk 1 above)
Comment (kurtseifried) [reply 2018/1000xxx/CVE-2018-1000853.json]: fixed

File: 2018/1000xxx/CVE-2018-1000859.json (hunk 2 above)
Comment (kurtseifried) [reply 2018/1000xxx/CVE-2018-1000859.json]: fixed

**Comment by kurtseifried** [comment]
This should be ready to go now.

**Pull request merged**